examples/titanic/simple_model_training.py

```diff
@@ -19,14 +19,15 @@
 # Define Ludwig model object that drive model training
 model = LudwigModel(config='./model1_config.yaml',
                     logging_level=logging.INFO)
+dataset = Titanic().load()
 
 # initiate model training
 (
     train_stats,  # dictionary containing training statistics
     preprocessed_data,  # tuple Ludwig Dataset objects of pre-processed training data
     output_directory # location of training results stored on disk
  ) = model.train(
-    dataset='./data/train.csv',
+    dataset=dataset,
```

Comment:
We probably only want the train split here.

Reply:
added